plutarch-extra/Plutarch/Extra/List.hs

```diff
@@ -1,7 +1,117 @@
-module Plutarch.Extra.List (preverse, pcheckSorted) where
+module Plutarch.Extra.List (
+  -- * Construction
+  preplicate,
+
+  -- * Transformation
+  pmapMaybe,
+  preverse,
+
+  -- * Search
+  pfindJust,
+  plookupAssoc,
+
+  -- * Elimination
+  precListLookahead,
+
+  -- * Check
+  pcheckSorted,
+) where
 
 import Plutarch.Prelude
 
+{- | Similar to 'pmap', but allows elements to be thrown out. More precisely,
+ for elements where the function argument returns 'PNothing', the
+ corresponding element is removed, while for elements where the function
+ argument returns `PJust`, the value is used in the result.
+-}
+pmapMaybe ::
+  forall (ell :: PType -> PType) (b :: PType) (a :: PType) (s :: S).
```

Comment:
ell?

Reply:
Like the capital letter of the alphabet. Calling to mind 'List'.